packages/fxa-auth-server/fxa-oauth-server/scripts/retire-old-signing-key.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/*
+
+ Usage:
+ scripts/retire-new-signing-key.js
+
+ Will zero out the old signing key in ./config/oldKey.json.
+
+ */
+
+const fs = require('fs');
+const path = require('path');
+const assert = require('assert');
+
+const keyPath = path.resolve(__dirname, '../config/key.json');
+const oldKeyPath = path.resolve(__dirname, '../config/oldKey.json');
+
+function main(cb) {
```

Comment:
Worth adding tests for these new scripts to ensure they don't bit rot? I know @philbooth has added script tests in the past for just that reason.

Reply:
Yep! Added.